src/CMakeLists.txt

```diff
@@ -4,18 +4,13 @@ add_compile_options(
   ${CXX_WARNING_ARGS}
 )
 
-add_library(
-    ${SERVICE_LIB}
-    STATIC
-    adbd-client.cpp
-    exporter.cpp
-    greeter.cpp
-    indicator.cpp
-    rotation-lock.cpp
-    usb-manager.cpp
-    usb-monitor.cpp
-    usb-snap.cpp
-)
+set (SERVICE_LIB_SOURCES exporter.cpp indicator.cpp rotation-lock.cpp)
+
+if (ENABLE_LOMIRI_FEATURES)
+    list (APPEND SERVICE_LIB_SOURCES greeter.cpp adbd-client.cpp usb-manager.cpp usb-monitor.cpp usb-snap.cpp)
```

Comment:
multi-line here, too. It's much better readable then, I guess.

Reply:
Amended